user/src/main/java/com/jumar/user/exceptions/OneOrMoreUserRequiredFieldsNullException.java

```diff
@@ -0,0 +1,12 @@
+package com.jumar.user.exceptions;
+
+import java.io.Serial;
+
+public class OneOrMoreUserRequiredFieldsNullException extends IllegalArgumentException{
+    @Serial
+    private static final long serialVersionUID = 2;
```

Comment:
we don't normally bother about the serialVersionUID unless it's a project standard / convention. it's just so much noise